OpenSim/ExampleComponents/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+
+file(GLOB INCLUDES *.h)
+file(GLOB SOURCES *.cpp)
+
+OpenSimAddLibrary(
+    KIT ExampleComponents
+    AUTHORS "Ajay_Seth-Michael_Sherman-Ayman_Habib"
+    LINKLIBS ${Simbody_LIBRARIES} osimCommon osimSimulation osimActuators osimAnalyses osimTools
```

Comment:
EDIT: I think this library only depends on osimCommon and osimSimulation (for Controller, etc.).